src/core/tab-navigation-base/tab-navigation-base/index.ts

```diff
@@ -264,6 +264,21 @@ export class TabNavigationBase extends View implements TabNavigationBaseDefiniti
         // overridden by inheritors
         return null;
     }
+    /**
+     * Gets the parent fragment manager from a fragment to be used in destroying or hiding it.
+     * @param fragment target fragment
+     * @returns the parent fragment manager or null if none exists.
+     */
+    public _getParentFragmentManagerFromFragment(fragment: androidx.fragment.app.Fragment) {
+        if (!fragment) {
+            return null;
+        }
+        try {
```

Comment:
quick question why do we need to try catch here?
is native throwing sometimes?

Reply:
when the fragment isn't attached to a fragment manager it throws instead of returning anything

Reply:
ok